### PR TITLE
Refactors part 6: Getting rid of graph nomenclature

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

File: docs/pages/overview/6-faq.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ Grakn uses an intuitive [schema](https://blog.grakn.ai/what-is-an-ontology-c5baa
...
...wledge-model.html#rule-and-sub-type-inference)) or discovery of inconsistencies in the data ([validation](https://grakn.ai/pages/documentation/the-fundamentals/grakn-knowledge-model.html#data-validation)). For this reason, the schema must be clearly defined before loading data into the In Grakn, the schema is the formal specification of all the relevant concepts and their meaningful associations in a given domain. It allows objects and relationships to be categorised into distinct types, and for generic properties of those types to be expressed. Specifying the schema enables [automated reasoning](https://en.wikipedia.org/wiki/Inference_engine) over the [represented knowledge](https://en.wikipedia.org/wiki/Knowledge_representation_and_reasoning), such as the extraction of implicit information from explicit data ([inference](https://grakn.ai/pages/documentation/the-fundamentals/grakn-knowledge-model.html#rule-and-sub-type-inference)) or discovery of inconsistencies in the data ([validation](https://grakn.ai/pages/documentation/the-fundamentals/grakn-knowledge-model.html#data-validation)). For this reason, the schema must be clearly defined before loading data into the knowledge base.
```
Comment (aelred) [inline docs/pages/overview/6-faq.md]: I think @fppt has entered a loop

File: grakn-dist/src/test/java/ai/grakn/dist/PhilosophersExampleIT.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public void testThereAreFourPeople() {
     }
 
     @Test
-    public void testCynicismIsInTheGraph() {
+    public void testCynicismIsInTheKB() {
```
Comment (aelred) [inline grakn-dist/src/test/java/ai/grakn/dist/PhilosophersExampleIT.java]: we all know that feeling

File: grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java [hunk 3]
```diff
@@ -66,13 +66,13 @@
 import static org.apache.commons.lang.exception.ExceptionUtils.getFullStackTrace;
 
 /**
- * A Graql shell session for a single client, running on one graph in one thread
+ * A Graql shell session for a single client, running on one tx in one thread
```
Comment (aelred) [inline grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java]: I think this one is "knowledge base"

File: docs/pages/overview/6-faq.md (hunk 1 above)
Comment (fppt) [inline docs/pages/overview/6-faq.md]: Ooops. I blame vim

File: grakn-dist/src/test/java/ai/grakn/dist/PhilosophersExampleIT.java (hunk 2 above)
Comment (fppt) [inline grakn-dist/src/test/java/ai/grakn/dist/PhilosophersExampleIT.java]: 😄

File: grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java (hunk 3 above)
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java]: hurray

**Comment by fppt** [comment]
@aelred re-review please

**Pull request merged**